.changeset/breezy-wombats-care.md

```diff
@@ -0,0 +1,8 @@
+---
+'@firebase/database': minor
```

Comment:
Why database?

Reply:
I had mentioned `database` because the upgrade of `grps-js` I think indirectly affected its version as well. But looking more closely, that probably isn't the case. I'll remove it.